### ISSUE TITLE
Failing `toolbar` test

### ISSUE THREAD

**Issue by howard-e** [w3c/aria-practices#3331]
There's a new failure with the [toolbar_toolbar: menuitemradio elements have aria-checked set test](https://github.com/w3c/aria-practices/blob/37c5fc7eb40285919cbd3cd14a71421d7c509a54/test/tests/toolbar_toolbar.js#L358:L397). This seems to be affecting new contributions since the start of this week:

<details>

<summary>toolbar_toolbar: menuitemradio elements have aria-checked set failure</summary>

```bash
...
not ok 97 - toolbar_toolbar › content/patterns/toolbar/examples/toolbar.html [data-test-id="toolbar-menuitemradio-aria-checked"]: menuitemradio elements have aria-checked set
  ---
    name: AssertionError
    details:
      'Rejected promise returned by test. Reason:': |-
        TimeoutError {
          remoteStacktrace: '',
          message: `Waiting for menu to open␊
          Wait timed out after 10174ms`,
        }
    message: Rejected promise returned by test
    at: Wait timed out after 10174ms
  ...
...
...
```

</details>

Examples:
* [minor version bump dependabot PR which should not affect that the test](https://github.com/w3c/aria-practices/pull/3321)
* [new spin button pattern PR which affects no toolbar-based content](https://github.com/w3c/aria-practices/pull/3328)
